app/views/admin/collections/show.html.erb

```diff
@@ -57,12 +57,14 @@ Unless required by applicable law or agreed to in writing, software distributed
         <% end %>
       </div>
       <div class="collection_poster">
+        <% is_depositor=can?(:read, Admin::Collection) && !can?(:manage, Admin::Collection) %>
```

Comment:
I don't have a strong opinion between inlining the cancancan checks below or centralizing them in this line.  Either way works for me.

Reply:
I'm okay with the inline checks, since they are not long.
```suggestion
```